.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+dist
```

Comment:
Please remove dist folder from gitignore list. As we are required to use a webpack in this project, the dist file is an auto-generated main folder that includes index.js and index.html.

Reply:
I don't thing it is a good practice to add these generated files from the `dist folder` in the repository since it is generated from the source code by webpack (when we run `npm run build`). 
Especially because these files will be overwritten every time we run webpack after changes has been made to the source files.
That is why I believe `dist/` should be in the `.gitignore` file, what do you think?